test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```

Comment:
- Please test.md folder should be in the .gitignore file and not in the Github repository. 👍